src/components/activity-page/activity-page-content.tsx

```diff
@@ -20,6 +21,9 @@ interface IProps {
   setNavigation: (refId: string, options: INavigationOptions) => void;
   pluginsLoaded: boolean;
   pageChangeNotification?: IPageChangeNotification;
+  readAloud?: boolean;
+  setReadAloud?: (readAloud: boolean) => void;
+  readAloudDisabled?: boolean;
```

Comment:
These 3 properties are added to multiple interfaces, so there are quite a few copies of this code.
Wouldn't it make sense to create some base interface that would include them?
I guess all these page-related components might have some elements in common, like this new toggle.